data/json/recipes/armor/legs.json

```diff
@@ -439,6 +439,7 @@
   {
     "result": "shorts_cargo",
     "type": "recipe",
+    "id_suffix": "from fabric",
```

Comment:
All but an anomalous "from_stockings". Should I underscore the rest or just get rid of that single underscore?